src/models/propulsion/FGEngine.cpp

```diff
@@ -200,19 +186,13 @@ bool FGEngine::Load(FGFDMExec *exec, Element *engine_element)
   // Call ModelFunctions loader
   FGModelFunctions::Load(engine_element, exec, to_string((int)EngineNumber));
 
-// Find and set engine location
-
-  local_element = parent_element->FindElement("location");
-  if (local_element)  location = local_element->FindElementTripletConvertTo("IN");
-//  else      cerr << "No engine location found for this engine." << endl;
-// Jon: The engine location is not important - the nozzle location is.
-
-  local_element = parent_element->FindElement("orient");
-  if (local_element)  orientation = local_element->FindElementTripletConvertTo("RAD");
-//  else          cerr << "No engine orientation found for this engine." << endl;
-// Jon: The engine orientation has a default and is not normally used.
+  // If engine location and/or orientation is supplied issue a warning since they
+  // are ignored. What counts is the location and orientation of the thruster.
+  if (parent_element->FindElement("location"))
+    cerr << "Engine location ignored, only thruster location is used." << endl;
 
-  SetPlacement(location, orientation);
+  if (parent_element->FindElement("orient"))
+    cerr << "Engine orientation ignored, only thruster orientation is used." << endl;
```

Comment:
```suggestion
  local_element = parent_element->FindElement("orient");
  if (local_element)
    cerr << local_element->ReadFrom()
         << "Engine orientation ignored, only thruster orientation is used." << endl;
```
Same as above.

Reply:
Pushed a commit to uses local_element->ReadFrom()